<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NSimSun</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string comment="Message">Message here...</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>NSimSun</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
         <underline>true</underline>
         <stylestrategy>PreferAntialias</stylestrategy>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>TOKEN*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="TokenEdit">
       <property name="font">
        <font>
         <family>NSimSun</family>
         <pointsize>15</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="SendBut">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>SEND</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="ChatBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>301</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="ConfigBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>167</width>
       <height>130</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="NameEdit">
        <property name="placeholderText">
         <string>Your Username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="MateIpEdit">
        <property name="placeholderText">
         <string>Enter Ip of your Mate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ConnectButton">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Connect </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="discButton">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="ClearBut">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Chat</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>565</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
